typed/typed.go

```diff
@@ -138,7 +138,8 @@ func (tv TypedValue) Compare(rhs *TypedValue) (c *Comparison, err error) {
 	return c, nil
 }
 
-// RemoveItems removes each provided list or map item from the value.
+// RemoveItems removes each provided path from the value, including matching child paths.
```

Comment:
> including matching child paths

That part of the sentence is not super clear. I think the word recursive would definitely remind me of `rm -r` as in "removes the path and all of its sub-path"?